liota/lib/transports/mqtt.py

```diff
@@ -165,24 +151,41 @@ def __init__(self, url, port, identity=None, tls_conf=None, qos_details=None, cl
         if self.clean_session:
             # If user passes client_id, it'll be used.  Otherwise, it is left to the underlying paho
             # to generate random client_id
-            self._paho_client = paho.Client(self.client_id, self.clean_session, self.userdata,
-                                            protocol=getattr(paho, self.protocol), transport=self.transport)
             log.info("clean_session is set to True")
```

Comment:
info or debug level logging? why do we only log about clean_session?

Reply:
Removed the logging about clean_session as it is not required. And provided a comment in README.